test/index.js

```diff
@@ -320,31 +324,32 @@ function time(options) {
  */
 
 function fix() {
-  ['tests', 'original', 'new'].forEach(function(dir) {
+  ['compiled_tests', 'original', 'new'].forEach(function(dir) {
     try {
-      fs.mkdirSync(path.resolve(__dirname, dir), 0755);
+      fs.mkdirSync(path.resolve(__dirname, dir), 0o755);
```

Comment:
`0o755` - it's not typo?

Reply:
That is an [octal literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Grammar_and_types#Integers)